packages/mesa.rb

```diff
@@ -45,8 +37,8 @@ def self.build
                 --prefix=#{CREW_PREFIX} \
                 --libdir=#{CREW_LIB_PREFIX} \
                 --enable-shared-glapi \
-                --with-gallium-drivers=i915,r300,r600,radeon,radeonsi,nouveau,svga,swrast,virgl \
-                --with-dri-drivers=i915,i965,nouveau,swrast,r200,radeon \
+                --with-gallium-drivers=i915,r300,r600,radeonsi,nouveau,svga,swrast,virgl \
+                --with-dri-drivers=i915,i965,nouveau,swrast,r200 \
```

Comment:
Just curious, do you know why dropping radeon fixes the package?

Reply:
What happens if we just remove these lines?  Does it autodetect the current drivers?  If that is the case, then specifying drivers is not necessary.

Reply:
No. The drivers for all Chromebooks are different. This is good.

Reply:
Is this driver list exhaustive?